app/views/forms/show.html.erb

```diff
@@ -7,8 +7,11 @@
     <h1 class="govuk-heading-l">
       <span class="govuk-caption-l"><%= @form.name %></span><span class="govuk-visually-hidden"> - </span>
       <%= @form.live? ? t("forms.form_overview.live_title") : t("forms.form_overview.title") %>
+      <span class="govuk-!-display-block"><%= render FormStatusTagComponent::View.new(status: @form.status) %></span>
```

Comment:
Not sure this should be inside the H1

Reply:
TBH, i think these headings could be converted to proper view components to keep them consistent.  I placed it inside to give it a bit more context otherwise its kind of a floating "LIVE"/"DRAFT" in the middle of no where with no context

Reply:
Yeah, see what you mean about wanting to give it some context, I just worry that we're starting to overstuff these H1s a bit.

Wonder if something like this would work:
```erb
<h1><%= whatever_h1_content %></h1>
<dl>
    <dt class="govuk-visually-hidden">Status</dt>
    <dd><%= render FormStatusTagComponent::View.new(status: @form.status) %></dd>
</dl>
```

That `dl` approach would be consistent with how Content Publisher and Whitehall mark up statuses, and we could probably add more metadata to that area in future if the design called for it.